.azure-pipelines/azure-pipelines-UpgrateVersion.yml

```diff
@@ -12,6 +12,8 @@ schedules:
   branches:
     include:
     - 202012
+    - '202[1-9]*'
```

Comment:
The 5th and 6th digits are the month. Let's pattern match them and keep string length fixed.

Reply:
The regex is not supported in AZP yet, it only supports ? and *.

Reply:
Fixed.